command/user.go

```diff
@@ -284,6 +308,24 @@ func randString(n int) string {
 	return string(bs)
 }
 
+func validCredentials(usr, pwd string) (bool, string) {
```

Comment:
In general terms in golang we should try to return an error instead of a bool and string. 

You could have something like:
```
func validateUsername(username string) error {
    if username != "" {
        return errors.New("Username can't be empty")
    }
    return
}
```

so on the other side you'll be consuming this method with something like:

```
if err := validateUsername(username); err != nil {
   log.Println(err.Error())
   return err
}
```

This becomes on a more standard and easy to read code.
I'll add full validation on these methods, i mean throwing errors in case the username is empty. So you have all validations on that field packed on the same method.